From a3ac51416813ee0f1601e06786b55d33447049ee Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 3 Sep 2008 16:47:42 +0800
Subject: [PATCH 018/256] ENGR000090137 fix MX37 VPU compile error

Fix MX37 VPU compile error due to MX51 VPU encoder check in.
Add MJPG_ENC definition in MX37 case.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/vpu_util.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/vpu/vpu_util.h b/vpu/vpu_util.h
index eb523df..d6f65e5 100644
--- a/vpu/vpu_util.h
+++ b/vpu/vpu_util.h
@@ -32,7 +32,8 @@ enum {
 	AVC_ENC = 0x7F,
 	MP4_ENC = 0x80,
 	RV_DEC = 0x81,
-	MJPG_DEC = 0x82
+	MJPG_DEC = 0x82,
+	MJPG_ENC = 0x83
 };
 #elif defined(IMX51_3STACK)
 enum {
@@ -58,7 +59,8 @@ enum {
 	DV3_DEC = 6,
 	/* dummy */
 	RV_DEC = 0x81,
-	MJPG_DEC = 0x82
+	MJPG_DEC = 0x82,
+	MJPG_ENC = 0x83
 };
 #endif
 
-- 
1.8.0

